### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Modifies the FAQ, the About page and the Play! page with up to date information.

Also adds a sixth button to round out the header design and list our donation link.

Also replaces the RSS link on the header with a subreddit link.

I think that's all I did i'm too tired to check.

### CODE REVIEW

File: content/about/faq/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 +++
 title = "Frequently Asked Questions"
-date = "2019-03-28"
+date = "Last Updated 6/9/20"
```
Comment (ComicIronic) [inline content/about/faq/_index.md:3]: Why would you switch from an unambiguous date format to an ambiguous one, and then pick the American version?

**Comment by PJB3005** [comment]
Conflicts.

File: content/about/faq/_index.md [hunk 2]
```diff
@@ -12,24 +12,24 @@ A: Nah.
...
 ### Q: Is There a Patreon? Can I donate?
 
-A: No. We have no methods of donating to the project at this time. We don't think it'd help to speed up development. While a system like bounties *might* be nice to encourage contribution, it's also a hassle and a potential source of drama.
+A: [Yes.](https://github.com/space-wizards/space-station-14) We started a Patreon in order to pay for productivity software, server hosting, work commissions, and more! In addition to providing rewards, we will also regularly post a financial statement that shows how our income is being used in the game's development. We want to give our patrons full trust and confidence, and we believe that being 100% transparent about how the money is being used is best.
 
-### Q: Will SS14 Go On Steam?
+### Q: Where can I play SS14?
 
-A: Yes! We're actively working on getting the store page ready for publishing. It'll obviously be free on steam too.
+A: You can download SS14 right now from our website, or wait for our free Steam release which we estimate will come out in Fall of 2020.
```
Comment (PJB3005) [inline content/about/faq/_index.md:35]: You had one job man.

Comment (SweptWasTaken) [reply content/about/faq/_index.md:35]: What's wrong with this

Comment (PJB3005) [reply content/about/faq/_index.md:35]: You could at least mention that the game isn't playable yet. i.e. only for testing.

**Pull request merged**